cms/static/js/views/paging_header.js

```diff
@@ -40,32 +46,40 @@ define(["underscore", "backbone", "gettext", "text!templates/paging-header.under
                     if (this.view.collection.sortDirection === 'asc') {
                         // Translators: sample result:
                         // "Showing 0-9 out of 25 total, filtered by Images, sorted by Date Added ascending"
-                        message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, filtered by %(asset_type)s, sorted by %(sort_name)s ascending');
+                        message = gettext(
+                            'Showing {currentItemRange} out of {totalItemsCount}, filtered by {assetType},' +
+                            ' sorted by {sortName} ascending'
+                        );
                     } else {
                         // Translators: sample result:
                         // "Showing 0-9 out of 25 total, filtered by Images, sorted by Date Added descending"
-                        message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, filtered by %(asset_type)s, sorted by %(sort_name)s descending');
+                        message = gettext(
+                            'Showing {currentItemRange} out of {totalItemsCount}, filtered by {assetType},' +
```

Comment:
Unfortunately you can't break ``gettext`` calls across multiple lines because the Transifex string scraper won't recognize them. You need to put this back on a single line and then use a jshint directive to ignore the extra long line.